docs/user-guide/sql/intro.md

```diff
@@ -63,6 +63,28 @@ We execute SQL queries by calling `execute` on a `SQLContext`.
 --8<-- "python/user-guide/sql/intro.py:execute"
 ```
 
+## Execute queries from multiple sources
+
+SQL queries can be executed just as easily from multiple sources.
+In the example below, we register :
+
+- a CSV file loaded lazily
+- a NDJSON file loaded lazily
+- a Pandas DataFrame
+
```

Comment:
Could you provide a sample of the file in the markdown such that people understand what is in them?

Reply:
@c-peters sorry for the late reply. just added an example of the schema. does that look ok?

Reply:
LGTM, Thanks!